src/tests/ftest/util/dmg_utils.py

```diff
@@ -49,18 +49,23 @@ class DmgCommand(CommandWithSubCommand):
         "pool_list": r"(?:([0-9a-fA-F-]+) +([0-9,]+))"
     }
 
-    def __init__(self, path):
+    def __init__(self, path, yaml_cfg=None):
```

Comment:
It's better to add description for yaml_cfg.

Reply:
Done.